<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDFS_Client</class>
 <widget class="QMainWindow" name="PDFS_Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PDFS_Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>450</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="SHost">
        <property name="text">
         <string>Host:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ServerHost"/>
      </item>
      <item>
       <widget class="QLabel" name="SState">
        <property name="text">
         <string>State:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ServerState">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ServerConnect">
        <property name="text">
         <string>connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="UserArea">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="UserNameLayout">
          <item>
           <widget class="QLabel" name="UserNameLabel">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>UserName:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="UserName"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="PasswordLayout">
          <item>
           <widget class="QLabel" name="PasswordLabel">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Password"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="Login">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="FileTree">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="DLPathLayout">
          <item>
           <widget class="QLabel" name="DLPathLabel">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>DLPath:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="DLPath"/>
          </item>
          <item>
           <widget class="QPushButton" name="Download">
            <property name="text">
             <string>Download</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="ULPathLayout">
          <item>
           <widget class="QLabel" name="ULPathLabel">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>ULPath:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ULPath"/>
          </item>
          <item>
           <widget class="QPushButton" name="UpLoad">
            <property name="text">
             <string>Upload</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="DebugArea">
        <item>
         <layout class="QHBoxLayout" name="DebugLayout">
          <item>
           <widget class="QLabel" name="DebugLabel">
            <property name="text">
             <string>Debug:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="DebugCommand"/>
          </item>
          <item>
           <widget class="QPushButton" name="DebugSend">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="DebugInfo"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="DownLoadLayout">
          <item>
           <widget class="QLabel" name="DownloadLabel">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>DownLoad:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="DownloadProgress">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="UploadLayout">
          <item>
           <widget class="QLabel" name="UploadLabel">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>UpLoad:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="UploadProgress">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>Login</tabstop>
  <tabstop>FileTree</tabstop>
  <tabstop>DebugCommand</tabstop>
  <tabstop>DebugSend</tabstop>
  <tabstop>DebugInfo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
